src/app/components/HomePage.tsx

```diff
@@ -20,29 +22,58 @@ const UniProtFooter = lazy(
 const mission =
   'UniProt is the world’s leading high-quality, comprehensive and freely accessible resource of protein sequence and functional information.';
 
-const HomePage = () => (
-  <>
-    <main>
-      <ErrorBoundary>
-        <HeroHeader title="Find your protein" footer={mission}>
-          <section className="uniprot-grid uniprot-grid--centered">
-            <SearchContainer className="uniprot-grid-cell--span-12" />
-          </section>
-        </HeroHeader>
-      </ErrorBoundary>
-
-      <ErrorBoundary>
-        <Suspense fallback={<Loader />}>
-          <HomePageNonCritical />
-        </Suspense>
-      </ErrorBoundary>
-    </main>
-    <Suspense fallback={null}>
-      <ErrorBoundary>
-        <UniProtFooter />
-      </ErrorBoundary>
-    </Suspense>
-  </>
-);
+// TODO remove "Partial" once all namespaces are added
```

Comment:
I think the comment doesn't apply as it's covering all the currently defined Namespaces, and the `Partial` should be removed.